### DISCUSSION TITLE
Auto-Animate in Markdown

### DISCUSSION THREAD

**Discussion by trochitv** [hakimel/reveal.js#3066] [Q&A]
Hey there!

I'd like to write a presentation in markdown (like you can with hackmd.io) and use the [auto-animate](https://revealjs.com/auto-animate/) functionality. Is this possible? If so, how?

Thanks in advance :)

**Comment by hakimel** [comment]
You can enable auto-animate using reveal.js slide attributes syntax—I've included an example below. This will only work with the built-in Markdown parser so I can't comment on if it's possible via other Markdown editors.

```html
<section data-markdown>
  <script type="text/template">
    <!-- .slide: data-auto-animate -->
    - One
    - Two
    - Three
    ---
    <!-- .slide: data-auto-animate -->
    - Three
    - Two
    - One
  </script>
</section>
```

**Comment by trochitv** [comment]
Thanks for the quick reply!
Unfortunately it did not work for me, but hopefully I'll find the time to learn html in the future.
